features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -114,6 +114,16 @@
     <bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-core-ephemeris" version="${project.version}">
```

Comment:
You miss the ephemeris bundle itself in this very specific feature

Reply:
Good point. No idea how that got lost...